### ISSUE TITLE
Static version of the logged in page

### ISSUE THREAD

**Issue by N-Yuki** [EaTea/CITS4230-Project-Group-7#6]
Write a skeleton mark-up for the logged in page.
Depends on GH-5.

**Comment by N-Yuki** [comment]
Can we get some status update on this or something? There's been no visible progress on this and it is close to the due date...

**Comment by N-Yuki** [comment]
Do not close issues that have not been resolved.
Unless it is suggested that the logged in page will be made in an ad hoc manner, this is a critical issue which needs to be solved before we can actually make progress on any ruby related stuff on that page.

**Comment by sontobgay** [comment]
Never meant to close the issue. Thought the day long coding session[mentioned in the mail] was to do something with the RAILS stuff. I feel the view of the logged-in page is the major chunk of the assigned project, and i feel it requires further deliberation among ourselves than one person assigning to it.

**Comment by EaTea** [comment]
I closed the issue. I thought we'd end up making a dynamic page very soon
anyway and the static version isn't worthwhile anymore.

If you feel it's necessary to make a static version still, reopen the issue.
I did mean to make a comment explaining my rationale. Sorry about that.

I didn't think that you'd be going with the latter version for coding up the
dynamic version but that's fine.

Sorry to Sonam --- I know you didn't close the issue, that was all me, no
need to apologise for that.

On 25 September 2011 19:49, sontobgay <
reply@reply.github.com>wrote:

> Never meant to close the issue. Thought the day long coding
> session[mentioned in the mail] was to do something with the RAILS stuff. I
> feel the view of the logged-in page is the major chunk of the assigned
> project, and i feel it requires further deliberation among ourselves than
> one person assigning to it.
> 
> ## 
> 
> Reply to this email directly or view it on GitHub:
> 
> https://github.com/EaTea/CITS4230-Project-Group-7/issues/6#issuecomment-2190621

## 

Edwin Tay

**Comment by N-Yuki** [comment]
I think there was a misunderstanding.

@sontobgay:
This issue wasn't to make the logged in page, but only just a static page based on the mockup - this should have been part of the stage 1 deliverable.

@EaTea:
Yes, we are making a dynamic page very soon, but I wasn't sure why we weren't doing a static version, and why this issue was closed. Unless you meant we write the static part along with the dynamic part - that's what I meant with the ad-hoc in my previous comment. I don't really mind which way we do it, I just wanted the issue resolved before being closed.

**Comment by EaTea** [comment]
My apologies. I thought that the issue was clearly communicated but
evidently there was a breakdown in communications somewhere.

There are benefits behind using the method we were initially planning (that
is, to create a static page and then add the dynamic parts in) but I thought
it would be double the work at this stage in the project and wouldn't be as
useful, hence why I closed the issue.

Is there a specific way to mark an issue as resolved? I didn't notice
anything, and I thought it would be good to mark the Stage 1 Milestone as
completed.

On 26 September 2011 00:35, S0uten <
reply@reply.github.com>wrote:

> I think there was a misunderstanding.
> 
> @sontobgay:
> This issue wasn't to make the logged in page, but only just a static page
> based on the mockup - this should have been part of the stage 1 deliverable.
> 
> @EaTea:
> Yes, we are making a dynamic page very soon, but I wasn't sure why we
> weren't doing a static version, and why this issue was closed. Unless you
> meant we write the static part along with the dynamic part - that's what I
> meant with the ad-hoc in my previous comment. I don't really mind which way
> we do it, I just wanted the issue resolved before being closed.
> 
> ## 
> 
> Reply to this email directly or view it on GitHub:
> 
> https://github.com/EaTea/CITS4230-Project-Group-7/issues/6#issuecomment-2191749

## 

Edwin Tay

**Comment by N-Yuki** [comment]
If this is ready, and you want me take the work in commit 4b34976ac29ce4b4bae977f8e58f24639c99cccc to apply it into the correct location in master, then I can do that.

**Comment by N-Yuki** [comment]
Well, EaTea has made a new mockup, and is building it up as it is being implemented using the base structure I wrote, so this issue is no longer relevant.

@EaTea:
I'll leave it up to you to decide if there are any parts in 4b34976 that you could use.
